buildconfig/config_emsdk.py

```diff
@@ -1,18 +1,18 @@
 """Config on Emscripten SDK is almost like Unix"""
 
-import os, sys
-from glob import glob
-import platform
 import logging
+import os
+import sys
+from glob import glob
+
 from distutils.sysconfig import get_python_inc
 
+from .config_unix import get_porttime_dep
```

Comment:
why is this extra import here?

Reply:
It is an _undefined name_ that can potentially raise `NameError` at runtime so let's avoid that.

Reply:
That condition that calls `get_porttime_dep` in this file does not actually execute at runtime... I think that codepath should be removed instead of adding this import

Reply:
I feel unqualified to make this change.

Reply:
There is a `if not is_wasm:` block guarding the `get_porttime_dep` function call... but at the top of the file there is also a systemexit happening for the same condition, so this can be removed safely.

If you wish, I could make a commit for it in this branch, or you could do it yourself too :)

Reply:
You removed that unused block correctly, but the import is still here so this needs to be removed too